### ISSUE TITLE
Virtual directory path mismatch

### ISSUE THREAD

**Issue by tomvanenckevort** [uComponents/nuPickers#120]
When running an Umbraco 7 (v7.3.1) with nuPickers inside a virtual directory (e.g. 'my-virtual-dir') it currently fails to load the nuPickers embedded resources.

When I look at the Application JS response which returns the list of JS files to lazyload it looks like this:

```
LazyLoad.js([
  "/my-virtual-dir/umbraco/lib/jquery/jquery.min.js?cdv=312276359",
  "/my-virtual-dir/umbraco/lib/angular/1.1.5/angular.min.js?cdv=312276359",
  "/my-virtual-dir/umbraco/lib/underscore/underscore-min.js?cdv=312276359",
  ...
  "/App_Plugins/nuPickers/Shared/DotNetDataSource/DotNetDataSourceConfigController.js.nu?cdv=312276359",
  "/App_Plugins/nuPickers/Shared/SaveFormat/SaveFormatResource.js.nu?cdv=312276359",
  "/App_Plugins/nuPickers/Shared/CustomLabel/CustomLabelConfigController.js.nu?cdv=312276359",
  ...
```

As you can see the virtual directory gets ignored for the nuPicker resources.

I think the issue here might be that the resource root URL as set in `EmbeddedResource.RootUrl` does not start with a tilde (`~`) which means that the virtual directory does not get prepended when generating the list for the lazyload JS files.

**Comment by calbuero** [comment]
Hi, I'm actually having this particular issue running Umbraco 7 with nuPickers inside a virtual directory. Any workaround to resolve this? Unfortunately we have to run Umbraco in a virtual directory for this particular project.

**Comment by roegierssven** [comment]
A tilde fixed the problem, at least in 7.4.3 and probably higher versions #147

**Comment by Hendy** [comment]
Unfortunately the fix breaks when debug mode is off (the inbound requests don't have the ~ to compare against) - a new fix might be to append the ~ in each property editor definition.
